assets/src/edit-story/components/keyboardShortcutsMenu/keyboardShortcutList.js

```diff
@@ -32,21 +32,43 @@ const optionOrAlt = isMacOs ? SPECIAL_KEYS.OPTION : SPECIAL_KEYS.ALT;
 const shortcuts = {
   header: {
     label: __('Keyboard Shortcuts', 'web-stories'),
-    shortcut: [cmdOrCtrl, '/'],
+    shortcut: (
+      <kbd>
+        <kbd aria-label={cmdOrCtrl.title}>{cmdOrCtrl.symbol}</kbd>
```

Comment:
💭 I don't know if this would help at all, but maybe this could work as a component?

```js
const idkNamingIsHard = (key) => <kbd aria-label={key.title}>{key.symbol}</kbd>
```

But it's like all one line lol so 🤷. I don't really mind the repetition though, so whatever works!